src/enqueue/class-localization-constants.php

```diff
@@ -26,12 +27,22 @@ class Localization_Constants implements Filters {
    * @param Manifest_Data $manifest           Inject manifest which holds data about assets from manifest.json.
    * @param Base_Route    $dynamics_crm_route Dynamics CRM route object which holds values we need to localize.
    */
-  public function __construct( Manifest_Data $manifest, Base_Route $dynamics_crm_route, Base_Route $buckaroo_ideal_route, Base_Route $buckaroo_emandate_route, Base_Route $send_email_route ) {
+  public function __construct(
+    Manifest_Data $manifest,
+    Base_Route $dynamics_crm_route,
+    Base_Route $buckaroo_ideal_route,
+    Base_Route $buckaroo_emandate_route,
+    Base_Route $send_email_route,
+    Base_Route $mailchimp_route,
```

Comment:
Why are you type-hinting against an abstract class and not an interface?

Reply:
Because there isn't really an interface for this (as everything is in `Base_Route`). Changed to `Callable_Route` instead.